.github/workflows/integration-informing.yaml

```diff
@@ -35,9 +54,6 @@ jobs:
       - name: Install snapcraft
         run: |
           sudo snap install snapcraft --classic
-      - name: Apply ${{ matrix.patch }} patch
-        run: |
-          ./build-scripts/patches/${{ matrix.patch }}/apply
```

Comment:
Should this file be deleted?

Reply:
i'd keep it _for now_, until things settle